### DISCUSSION TITLE
How to disable drop only for specific elements

### DISCUSSION THREAD

**Discussion by omergr8** [GrapesJS/grapesjs#4065] [Q&A] [answered]
Hi, I need to know how can I set droppable to false only when some specific type of element is being dropped.
I have followed the following instruction:
droppable: '[data-gjs-type="column"], [data-gjs-type="other"]',
but the above approach only allow the mentioned elements to be dropped.
I want something like, everything is droppable except data-gjs-type="text"
Thank you

**Comment by artf** [comment, marked as answer]
Did you try with something like `droppable: ':not([data-gjs-type="text"])'`?

**Reply by omergr8** [reply]
Thank you, it worked.
